### ISSUE TITLE
Breaks in Browser due to process

### ISSUE THREAD

**Issue by mattdesl** [devongovett/jpg-stream#2]
The compiled output currently breaks in browserify. Offending line:

```js
Module["thisProgram"]=process["argv"][1].replace(/\\/g,"/")

   Uncaught TypeError: Cannot read property 'replace' of undefined
```

It also relies on other features like `process.stdout` etc.

**Comment by devongovett** [comment]
Yeah browserify and emscripten don't get along very well. You need to tell browserify to not parse that file. Unfortunately, I don't have a way to do that at the module level, so every project using jpg-stream (or any other emscripten project) must do that.

```
browserify --noParse jpg-stream/build/jpeg.js ...
```

**Comment by mattdesl** [comment]
Cool, thanks.
